doc/de/Channels.md

```diff
@@ -0,0 +1,99 @@
+Kanäle (Channels)
+=====
+
+* [Home](help)
+
+Kanäle sind eine Möglichkeit neue Inhalte zu finden, oder Inhalte anzuzeigen, die du sonst möglicherweise verpasst hättest.
+Es gibt mehrere vordefinierte Kanäle und zusätzlich kannst du deine eigenen, basierend auf ein paar Regeln, erstellen.
+Kanäle zeigen nur Beiträge aus den letzten 24 Stunden an. (Dieser Wert kann vom Administrator geändert werden.)
+
+In den Anzeige-Einstellungen, im Bereich "Timelines", kannst du definieren, welche Kanäle und andere Timelines du im "Kanäle"-Widget auf der "Network"-Seite sehen möchtest und welche Kanäle in der Menüleiste oben auf der Seite erscheinen sollen.
```

Comment:
I think it would be more clear to write the complete path here "Einstellungen -> Anzeige -> Timelines" here. Likely the same for the EN original of the docs.